src/lake/Lake/Config/Package.lean

```diff
@@ -310,9 +319,13 @@ namespace Package
 @[inline] def precompileModules (self : Package) : Bool :=
   self.config.precompileModules
 
-/-- The package's `moreServerArgs` configuration. -/
-@[inline] def moreServerArgs (self : Package) : Array String :=
-  self.config.moreServerArgs
+/-- The package's `moreGlobalServerArgs` configuration. -/
+@[inline] def moreGlobalServerArgs (self : Package) : Array String :=
+  self.config.moreGlobalServerArgs
+
+/-- The package's `moreServerOptions` configuration. -/
+@[inline] def moreServerOptions (self : Package) : Array LeanOption :=
+  self.config.moreServerOptions
```

Comment:
```suggestion
  self.leanOptions ++ self.config.moreServerOptions
```

Reply:
In this case, it should probably be named `serverOptions`, though.